rio_cogeo/scripts/cli.py

```diff
@@ -258,24 +262,33 @@ def create(
         output_profile["blockxsize"] = blocksize
         output_profile["blockysize"] = blocksize
 
-    if web_optimized:
-        overview_blocksize = blocksize or 512
-
     config.update(
         {
             "GDAL_NUM_THREADS": threads,
             "GDAL_TIFF_INTERNAL_MASK": os.environ.get("GDAL_TIFF_INTERNAL_MASK", True),
-            "GDAL_TIFF_OVR_BLOCKSIZE": str(overview_blocksize),
         }
     )
 
+    tilematrixset: TileMatrixSet | None
     if tms:
         with open(tms, "r") as f:
             tilematrixset = TileMatrixSet(**json.load(f))
-
+    elif web_optimized:
+        tilematrixset = morecantile.tms.get("WebMercatorQuad")
     else:
         tilematrixset = None
 
+    gdal_tif_ovr_blocksize: int | None
```

Comment:
`gdal_tif_ovr_blocksize` should never be none, and at least should default to 128

Reply:
Related to https://github.com/cogeotiff/rio-cogeo/pull/279#discussion_r1466095298